de-DE/Common.js

```diff
@@ -0,0 +1,13 @@
+export default {
+  'Navigation': {
```

Comment:
Are the following correct German translations?

- Sessions
- Web Sources
- "Tools"
- About
-

Reply:
These words are regularly used as so-called anglicisms in the German language in this context. There are existing German translations, but those would sound strange even to native speakers as myself.